HotelBooking/Database/create_database.sql

```diff
@@ -12,25 +13,50 @@ CREATE TABLE administrator
 CREATE TABLE room
 (
   room_id VARCHAR(20) NOT NULL PRIMARY KEY,
-  room_type TEXT CHECK(room_type IN ('SINGLE', 'DOUBLE', 'DELUXE', 'PRESIDENTIAL')) NOT NULL,
-  room_status TEXT CHECK(room_status IN('AVAILABLE', 'CHECKED-IN', 'RESERVED')) NOT NULL DEFAULT 'AVAILABLE',
-  customer_id INTEGER,
-  status_time TEXT,
-  FOREIGN KEY(customer_id) REFERENCES customer(customer_id)
+  room_type TEXT NOT NULL,
```

Comment:
why did you remove `CHECK(room_type IN ('SINGLE', 'DOUBLE', 'DELUXE', 'PRESIDENTIAL'))` ?

Reply:
because room-type do not have to be in 'SINGLE', 'DOUBLE', 'DELUXE', 'PRESIDENTIAL'

Reply:
then you should keep this check in the `roomType` table as it is being foreignly referrenced.